cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -138,7 +138,7 @@ func (uc *UpstreamController) Start() error {
 	go uc.dispatchMessage()
 
 	for i := 0; i < int(uc.config.Load.UpdateNodeStatusWorkers); i++ {
-		go uc.updateNodeStatus()
+		go uc.patchNode()
```

Comment:
We already have a `patchNode` at L168, this one is used to keep compatible with old version of edgecore. And will be removed in future release. @cl2017

Reply:
ok, I'll revert this change